cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1297,14 +1323,20 @@ def make_resource(self, data, **kwargs):
 class LoginNodesPoolSchema(BaseSchema):
     """Represent the schema of the LoginNodesPool."""
 
-    name = fields.Str(required=True)
-    instance_type = fields.Str(required=True)
-    image = fields.Nested(LoginNodesImageSchema)
-    networking = fields.Nested(LoginNodesNetworkingSchema, required=True)
-    count = fields.Int(required=True, validate=validate.Range(min=0))
-    ssh = fields.Nested(LoginNodesSshSchema, required=True)
-    iam = fields.Nested(LoginNodesIamSchema)
-    gracetime_period = fields.Int(validate=validate.Range(min=1, max=120))
+    name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    instance_type = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    image = fields.Nested(LoginNodesImageSchema, metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    networking = fields.Nested(
+        LoginNodesNetworkingSchema, required=True, metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP}
+    )
+    count = fields.Int(
+        required=True, validate=validate.Range(min=0), metadata={"update_policy": UpdatePolicy.SUPPORTED}
```

Comment:
Should set default value to `0` with `missing=0`

Reply:
Default values are not set in the schema but at the config resource level.
Currently it is set to 1 and it shoul dbe 0 so i will fix in the next iteration.